Kulipa.Sdk/Models/Users/Phone.cs

```diff
@@ -1,22 +1,26 @@
+using System.ComponentModel.DataAnnotations;
 using System.Text.Json.Serialization;
 
 namespace Kulipa.Sdk.Models.Users
```

Comment:
Should we maybe not work with the request and response models so it's more easy so distinguish them?